packages/base/card-api.gts

```diff
@@ -220,9 +222,10 @@ const loadLinksPromises = initSharedState(
   'loadLinksPromises',
   () => new WeakMap<BaseDef, Promise<any>>(),
 );
-const identityContexts = initSharedState(
-  'identityContexts',
-  () => new WeakMap<BaseDef, IdentityContext>(),
+// TODO do we really need this???
+const stores = initSharedState(
+  'stores',
+  () => new WeakMap<BaseDef, CardStore>(),
```

Comment:
look into this more...

Reply:
this seems related to how we deal with the NotLoaded errors. I'll dive more into the need for this in CS-9036